src/App.tsx

```diff
@@ -1,12 +1,60 @@
 import React from 'react';
 import './App.scss';
+import { GoodsList } from './components/GoodsList/GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+type State = {
+  goods?: Good[],
+};
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component<State> {
+  state = {
+    goods: [],
+  };
+
+  getList = (getSample:any) => {
+    getSample()
+      .then((result:any) => {
```

Comment:
```suggestion
      .then((result: Good[]) => {
```
Избегай использования any